packages/tokens/src/color/darkModeColors.ts

```diff
@@ -0,0 +1,159 @@
+import { palette } from '@leafygreen-ui/palette';
+
+import { MapVariantsToStates, State, Type, Variant } from './color.types';
+
+const { black, blue, gray, green, red, white, yellow } = palette;
+
+const darkModeBackgroundColors: MapVariantsToStates = {
+  [Variant.Primary]: {
+    [State.Default]: black,
+    [State.Hover]: gray.dark2,
+    [State.Focus]: blue.dark3,
+  },
+  [Variant.Secondary]: {
+    [State.Default]: gray.dark4,
+    [State.Hover]: gray.dark2,
+    [State.Focus]: blue.dark3,
+  },
+  [Variant.Inverse]: {
+    [State.Default]: gray.light2,
+    [State.Hover]: gray.light3,
+    [State.Focus]: blue.light2,
+  },
+  [Variant.Info]: {
+    [State.Default]: blue.dark3,
+    [State.Hover]: blue.dark3,
+    [State.Focus]: blue.dark3,
+  },
+  [Variant.Warning]: {
+    [State.Default]: yellow.dark3,
+    [State.Hover]: yellow.dark3,
+    [State.Focus]: yellow.dark3,
+  },
+  [Variant.Success]: {
+    [State.Default]: green.dark3,
+    [State.Hover]: green.dark3,
+    [State.Focus]: green.dark3,
+  },
+  [Variant.Error]: {
+    [State.Default]: red.dark3,
+    [State.Hover]: red.dark3,
+    [State.Focus]: red.dark3,
+  },
+  [Variant.Disabled]: {
+    [State.Default]: gray.dark2,
+    [State.Hover]: gray.dark2,
+    [State.Focus]: gray.dark2,
+  },
+};
+
+const darkModeBorderColors: MapVariantsToStates = {
+  [Variant.Primary]: {
+    [State.Default]: gray.base,
+    [State.Hover]: gray.base,
+    [State.Focus]: blue.light1,
+  },
+  [Variant.Secondary]: {
+    [State.Default]: gray.dark2,
+    [State.Hover]: gray.dark2,
+    [State.Focus]: blue.light1,
+  },
+  success: {
```

Comment:
some of the keys in this object and the objects below need to get swapped to the enum

Reply:
ah! thank you!